test/test_index_string.cpp

```diff
@@ -1876,7 +1876,6 @@ std::string create_random_a_string(size_t max_len) {
 
 }
 
-
```

Comment:
Please refrain from making unnecessary (white space) changes. Particularly in unrelated files.